hvplot/pandas.py

```diff
@@ -10,9 +10,15 @@ def patch(name='hvplot', extension='bokeh', logo=False):
                           'Pandas could not be imported.')
     _patch_plot = lambda self: hvPlotTabular(self)
     _patch_plot.__doc__ = hvPlotTabular.__call__.__doc__
-    patch_property = property(_patch_plot)
-    setattr(pd.DataFrame, name, patch_property)
-    setattr(pd.Series, name, patch_property)
+    plot_prop = property(_patch_plot)
+    setattr(pd.DataFrame, name, plot_prop)
+    setattr(pd.Series, name, plot_prop)
+
+    _patch_interactive = lambda self: Interactive(self)
+    _patch_interactive.__doc__ = Interactive.__call__.__doc__
+    interactive_prop = property(_patch_plot)
+    setattr(pd.DataFrame, name, interactive_prop)
+    setattr(pd.Series, name, interactive_prop)
```

Comment:
Is there a supported way to make the interactive support available without `import hvplot.pandas` or similar sideeffect-y imports?  Personally, I'll use `import hvplot.pandas`, but I'm curious if there is more awkward but less magic alternative, either to explicitly perform the monkeypatching or to have a non-monkeypatch alternative (e.g. a global function instead of a method).

Reply:
`Interactive(ds)` will work, I can also provide a helper to do the monkey-patching manually.